File: backend_laravel/app/Models/Employe.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use App\Models\User;
use App\Models\Tache;

class Employe extends Model
{
    protected $fillable = [
        'user_id',
        'competences',
        'niveau',
        'charge_actuelle',
    ];

    // Un employé appartient à un utilisateur
    public function user()
    {
        return $this->belongsTo(User::class);
    }

    // Un employé peut avoir plusieurs tâches
    public function taches()
    {
        return $this->hasMany(Tache::class);
    }
}

File: project_flutter/lib/models/employe.dart
class Employe {
  final int id;
  final int userId;
  final String nom;
  final String competences;
  final int niveau;
  final int chargeActuelle;

  Employe({
    required this.id,
    required this.userId,
    required this.nom,
    required this.competences,
    required this.niveau,
    required this.chargeActuelle,
  });

  factory Employe.fromJson(Map<String, dynamic> json) {
    return Employe(
      id: json['id'],
      userId: json['user_id'],
      nom: json['user']['name'],
      competences: json['competences'],
      niveau: json['niveau'],
      chargeActuelle: json['charge_actuelle'],
    );
  }
}